.github/workflows/ci-linux.yml

```diff
@@ -19,37 +19,37 @@ env:
 jobs:
 
   build:
-    name: CI on ${{ matrix.tag }}
+    name: CI on Linux
 
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        tag: # Those are our dockerhub alire/gnat:tag machines
-            - centos-latest-community-latest
-            - community-latest
-            - debian-stable
-            - ubuntu-lts
+    runs-on: ubuntu-18.04
 
     steps:
     - name: Check out repository
       uses: actions/checkout@v2
       with:
         submodules: true
 
-    - name: Pull docker image
-      run: docker pull alire/gnat:${{ matrix.tag }}
+    - name: Install toolchain
+      uses: ada-actions/toolchain@ce2020
+      with:
+        distrib: community    
```

Comment:
Why not the FSF one here? I would expect that [some] users building from source could be using that one. And since the CE is already tested on windows/macos...

Reply:
OK I can change that.